<launch>
    <arg name="command_topic" default="/picarx/drive/command"/>
    <arg name="motor_left_topic" default="/picarx/motor_left/speed"/>
    <arg name="motor_right_topic" default="/picarx/motor_right/speed"/>
    <arg name="steering_topic" default="/picarx/steer"/>


  <!-- convert joint states to TF transforms for rviz, etc -->
  <node name="ackermann_skill" pkg="picarx_ackermann_drive" type="ackermann_drive_skill.py"
    respawn="false" output="screen">
        <param  name="command_topic" type="str" value="$(arg command_topic)" />
        <param  name="motor_left_topic" type="str" value="$(arg motor_left_topic)" />
        <param  name="motor_right_topic" type="str" value="$(arg motor_right_topic)" />
        <param  name="steering_topic" type="str" value="$(arg steering_topic)"/>
  </node>
  
</launch>